webpack/scenes/Subscriptions/components/SubscriptionsTable/EntitlementsInlineEditFormatter.js

```diff
@@ -34,20 +34,28 @@ export const entitlementsInlineEditFormatter =
       );
     },
     renderEdit: (value, additionalData) => {
-      const { availableQuantity, availableQuantityLoaded } = additionalData.rowData;
+      const remainingQuantity = additionalData.rowData.availableQuantity;
+      const remainingQuantityLoaded = additionalData.rowData.availableQuantityLoaded;
+      const currentQuantity = additionalData.rowData.available;
```

Comment:
I still don't see the object destructuring with renaming [I mentioned here](https://github.com/Katello/katello/pull/7818#discussion_r235099192)?

You can do this:
```javascript
const { availableQuantity: remaningQuantity, availableQuantityLoaded: remainingQuantityLoaded, available: currentQuantity } = additionalData.rowData
```

I am curious if these can be renamed higher up the call stack? Where do these variables get declared? I can help you look if you would like to pair up on it. The camelcase suggests its coming from the JS side of things